Cargo.toml

```
@@ -24,8 +25,8 @@ all-algorithms = ["brotli", "bzip2", "deflate", "gzip", "lzma", "xz", "zlib", "z
 # algorithms
 deflate = ["flate2"]
 gzip = ["flate2"]
-lzma = ["xz2"]
-xz = ["xz2"]
+lzma = ["liblzma"]
```

```suggestion
lzma = ["deps:liblzma"]
```

We don't want any implicit features defined